src/UI/templates/default/Glyph/tpl.glyph.html

```diff
@@ -0,0 +1,80 @@
+<a class="glyph<!-- BEGIN highlighted --> highlighted<!-- END highlighted -->" <!-- BEGIN with_action -->href="{ACTION}"<!-- END with_action --> aria-label="{LABEL}">
+
+
+<!-- BEGIN settings -->
+<span class="glyphicon glyphicon-cog" aria-hidden="true"></span>
+<!-- END settings -->
+
+<!-- BEGIN collapse -->
+<span class="glyphicon glyphicon-triangle-bottom" aria-hidden="true">Issue: New Bootstrap Version (3.3.6) Needed</span>
+<!-- END collapse -->
+
+<!-- BEGIN expand -->
+<span class="glyphicon glyphicon-triangle-right" aria-hidden="true">Issue: New Bootstrap Version (3.3.6) Needed</span>
+<!-- END expand -->
+
+<!-- BEGIN add -->
+<span class="glyphicon glyphicon-plus-sign" aria-hidden="true"></span>
+<!-- END add -->
+
+<!-- BEGIN remove -->
+<span class="glyphicon glyphicon-minus-sign" aria-hidden="true"></span>
+<!-- END remove -->
+
+<!-- BEGIN up -->
+<span class="glyphicon glyphicon-circle-arrow-up" aria-hidden="true"></span>
+<!-- END up -->
+
+<!-- BEGIN down -->
+<span class="glyphicon glyphicon-circle-arrow-down" aria-hidden="true"></span>
+<!-- END down -->
+
+<!-- BEGIN back -->
+<span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+<!-- END back -->
+
+<!-- BEGIN next -->
+<span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+<!-- END next -->
+
+<!-- BEGIN sortAscending -->
+<span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span>
+<!-- END sortAscending -->
```

Comment:
Thank you for the feedback.

The KS will provide information on the less variables used. The classes are only indirectly provided in the example that is rendered for the component.

From my point of view, there are 4 levels of customizing to look of an ILIAS-Installation:

Level 1, Less Variables: 
- Every Admin is able to do that if basic css knowledge is available (meaning of px, padding etc.)
- Possible through frontend
- Limited by the scope of less variables

Level 2, Less Files:
- Deeper less/css knowledge required
- Access to CSS/Less files required
- More customizing option available
- Limited by the structure of the tpl.name.html files

Level 3, html Files:
- DOM knowledge required
- Superficial Templating Engine knowledge required
- Limited only by PHP-Variables

Level 4, Patch/Plugin etc.:
- PHP/ILIAS Knowledge Required
- Unlimited

I think designers need a fair chance to perform customization up to and including level 3. Why do we have the possibilities to customize tpl files if not for that? If classes are shiped by variables, you make it virtually impossible to change a class in the template without changing PHP-code.

Do you see limitation the pattern proposed by @klees  will bring (seperate section for each glyph)? The resulting redundancy in code might be problematic. This can be tackled in some cases however by using the slightly different pattern I used in the very top of this file the add the "highlighted class" if needed.